src/login.js

```diff
@@ -81,7 +81,10 @@ export const auth = {
 					const creds = await this.auth.loadCreds();
 					user.keys = creds.keys || {}; // preserve any exisiting keys
 				} catch (e) {
-					// skip when login is invoked for the first time and creds.json doesn't exist
+					// do nothing when login is invoked for the first time and creds.json doesn't exist
+					if (!e.includes('creds not found')) {
```

Comment:
Sorry... not to keep this going... but... I think this may cause an error for errors that are error objects...

Maybe something like:

```
if (e != 'creds not found') {
  console.log(chalk.red(e));
}
```